helloworld.js

```diff
@@ -0,0 +1,23 @@
+var express = require('express')
```

Comment:
We need to establish our coding styles for the project; I would prefer if we used mostly double quotation marks, and dropped our curly braces on new lines. That's just me, though, so if you disagree, I am definitely open to discussion.